plugins/request-uri-manipulations.rst

```diff
@@ -0,0 +1,25 @@
+Request URI Manipulations
+=========================
+
+Request URI manipulations can be done thanks to several plugins:
+
+* ``AddHostPlugin``: Sets default host, scheme and port of each request.
+* ``AddPathPlugin``: Sets default base path of each request.
```

Comment:
this is not a default but unconditionally applied: "Prefix the request path with a path, leaving the host information untouched."